Sources/Datadog/Tracing/Span/SpanEncoder.swift

```diff
@@ -0,0 +1,150 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+/// `Encodable` representation of span.
+internal struct Span: Encodable {
```

Comment:
can we rename it to something like `PersistedSpan`, `EncodableSpan` or `FinishedSpan`?

Reply:
Hmm, I don't like this idea from the architectural glance:

```swift
Log
LogBuilder
LogEncoder
LogSanitizer
```

```swift
PersistedSpan // 😬
SpanBuilder
SpanEncoder
SpanSanitizer  // eventually if we need to sanitize tags
```

WDYT?

Reply:
We asked other mobile devs for opinion and the verdict is `Span`
resolved ✅